### PR TITLE
Fixes Code Climate ES lint build failing

### PR DESCRIPTION
**Pull request by Royal-lobster**
# DRAFT

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   extends: './configs/eslint-next.js',
-  ignorePatterns: ['src/stories/*'],
+  ignorePatterns: ['/src/stories/*'],
```
Comment (kesar) [inline .eslintrc.js:3]: we dont even have stories

Comment (Adebesin-Cell) [reply .eslintrc.js:3]: I removed it 😅, it threw another error.

Comment (Royal-lobster) [reply .eslintrc.js:3]: Oh true, will add a commit for fix.

Comment (Adebesin-Cell) [reply .eslintrc.js]: Updated, let's see.

Comment (Adebesin-Cell) [reply .eslintrc.js]: Not sure, maybe we remove the yarn lock and node_modules, install them again? @Royal-lobster

**Comment by Adebesin-Cell** [comment]
Screen Shot 2023-02-08 at 9 09 44 AM

@kesar @Royal-lobster After changing the eslint-version to 8 in .codeclimate.yml, says here we have conflicting configs in eslint-next.js and .eslintrc.js

**Comment by Royal-lobster** [comment]
We are left with only one error with eslint-7 after the support recommendation. 

CleanShot 2023-02-08 at 19 06 06@2x

So we are using `eslint-plugin-unused-imports` plugin for checking unused imports which i believe is not yet supported on code climate. 


CleanShot 2023-02-08 at 19 07 22@2x


They are only checking with specific es lint versions and only with few subset of available eslint plugins

CleanShot 2023-02-08 at 19 05 53@2x

We checked the github repo from code climate but not able to find the plugin in their package json - https://github.com/codeclimate/codeclimate-eslint/blob/channel/eslint-7/package.json

CleanShot 2023-02-08 at 19 07 06@2x

**Comment by Royal-lobster** [comment]
Tried removing that module, we think it is not the case 😅. we still have this error we assume it is coming from their files from the stacktrace. we are still waiting for their response for the second error

CleanShot 2023-02-08 at 19 23 53@2x

**Comment by Royal-lobster** [comment]
@Adebesin-Cell did we get any further response from the support?

**Comment by Adebesin-Cell** [comment]
> @Adebesin-Cell did we get any further response from the support?

Nothing yet, I’ll have to send a reminder mail today.

**Comment by Adebesin-Cell** [comment]
@Royal-lobster, Any update from the codeclimate team?

**Comment by Royal-lobster** [comment]
> @Royal-lobster, Any update from the codeclimate team?

nope :/

**Comment by Adebesin-Cell** [comment]
> > @Royal-lobster, Any update from the codeclimate team?
> 
> nope :/

😅

**Comment by Adebesin-Cell** [comment]
Strong rejection 😅 @Royal-lobster. Possibly we remove and install the package again?

**Comment by Royal-lobster** [comment]
> Strong rejection 😅 @Royal-lobster. Possibly we remove and install the package again?

Yeah :(

**Comment by Royal-lobster** [comment]
We are getting this error even tho we removed ignorePatterns from eslint. it should work from eslint version >6 so i assume there is another eslint version running somewhere in the build process similar to this https://github.com/dice-roller/rpg-dice-roller/issues/122

CleanShot 2023-02-20 at 14 13 38@2x

**Comment by Adebesin-Cell** [comment]
> We are getting this error even tho we removed ignorePatterns from eslint. it should work from eslint version >6 so i assume there is another eslint version running somewhere in the build process similar to this [dice-roller/rpg-dice-roller#122](https://github.com/dice-roller/rpg-dice-roller/issues/122)
> 
> CleanShot 2023-02-20 at 14 13 38@2x

Yeah! I noticed that. Once can you test the codeclimate installation on the ep-template, let’s see if it works on a new project. It’s possible one of the dependencies has eslint running under the hood.

**Comment by Royal-lobster** [comment]
> Yeah! I noticed that. Once can you test the codeclimate installation on the ep-template, let’s see if it works on a new project. It’s possible one of the dependencies has eslint running under the hood.

Good point ! i think it will yield the same error but lets see

**Comment by Royal-lobster** [comment]
CleanShot 2023-02-20 at 20 50 06@2x

It is working locally

**Comment by Adebesin-Cell** [comment]
Ha! The issue is from their configs then @codeclimate.

**Comment by Royal-lobster** [comment]
CleanShot 2023-02-20 at 20 56 39@2x

This is fine too

**Comment by kesar** [comment]
if issue its on their side. I would recommend to stop working on this and remove the PR integration. We can still check the reviews on codeclimate until they fix it, but we wont get error in all our PRs because of this missconfiguration 👍🏻

**Comment by Adebesin-Cell** [comment]
> if issue its on their side. I would recommend to stop working on this and remove the PR integration. We can still check the reviews on codeclimate until they fix it, but we wont get error in all our PRs because of this missconfiguration 👍🏻

Yeah! I think we can remove the codeclimate integration on the repo for now. We’ll continue the review but in the meantime we can remove the plug-in to avoid errors on PRs. 🙏

**Comment by kesar** [comment]
cool! its deactivated for now 👍🏻

**Comment by Adebesin-Cell** [comment]
> cool! its deactivated for now 👍🏻

Awesome 🚀.

**Pull request closed without merging**